Disable auto conversion of sub-classes of list, dict, and tuple

youralmight opened the pull request:
Update __init__.py so that EasyDict does not convert subclasses of dict, tuple, and list like datasets.dataset_dict.DatasetDict

youralmight committed:
Update __init__.py so EasyDict does not convert subclasses of dict, tuple, and list like datasets.dataset_dict.DatasetDict

leplatrem reviewed (commented):
Could you please add a test to demonstrate your changes? 
Thank you

youralmight committed:
Create test.py

Add `test.py` to validate the auto-conversion of `list`, `tuple`, and `dict`, and to validate the non-auto-conversion of their subclasses.

youralmight committed:
Add `test.py` to validate the auto-conversion of `list`, `tuple`, and `dict`, and to validate the non-auto-conversion of their subclasses.

youralmight commented:
I added unittest to validate it.

youralmight commented:
@leplatrem

youralmight commented:
@leplatrem
